app/models.py

```diff
@@ -2038,18 +2034,19 @@ class DateTimeDimension(db.Model):
 class FactNotificationStatus(db.Model):
     __tablename__ = 'ft_notification_status'
 
-    bst_date = db.Column(db.Date, index=True, primary_key=True, nullable=False)
-    template_id = db.Column(UUID(as_uuid=True), primary_key=True, index=True, nullable=False)
+    bst_date = db.Column(db.Date, index=True, primary_key=True, nullable=False, default=datetime.datetime.utcnow)
+    template_id = db.Column(UUID(as_uuid=True), primary_key=True, index=True, nullable=False, default=uuid.uuid4)
```

Comment:
I'm skeptical that this is the correct approach.  The given template_id must be one that actually exists in the database.  Otherwise, trying to persist a new row will raise IntegrityError.  Using a default value doesn't make sense.

Reply:
One second look, IntegrityError will not get raised because the column isn't declared as a foreign key, but it still doesn't make sense to use a random value.

Reply:
Eh, I'm resolving this based on the text of the warning message.